client_plugin/drivers/shared/shared_driver.go

```diff
@@ -111,7 +147,109 @@ func (d *VolumeDriver) UnmountVolume(name string) error {
 
 // Create - create a volume.
 func (d *VolumeDriver) Create(r volume.Request) volume.Response {
-	log.Errorf("VolumeDriver Create to be implemented")
+	log.Infof("Entering VolumeDriver Create")
+	var msg string
+	var hnames []string
+
+	// Ensure that the node running this command is a manager
+	info, err := d.dockerd.Info(context.Background())
+	if err != nil {
+		msg = fmt.Sprintf("Failed to get Info from docker client. Reason: %v", err)
+		log.Warningf(msg)
+		return volume.Response{Err: msg}
+	}
+	if info.Swarm.ControlAvailable == false {
+		msg = fmt.Sprintf("This node is not a swarm manager. Shared Volume creation is only possible from swarm manager nodes.")
```

Comment:
It's possible to list/inspect volumes from all the nodes.
It's possible to support a volume created by vmdk plugin to share it through shared plugin. But this is not a planned feature for this release.